combine_avlqc.py

```diff
@@ -1,12 +1,43 @@
 #!/usr/bin/env python
 
-from os.path import join as pjoin, dirname
+from os.path import join as pjoin, dirname, isdir
 from os import makedirs
 import sys
 from glob import glob
 import pandas as pd
 import numpy as np
 
+def get_score(zipped_list):
+
+    # 5 excellent (<1% inaud),
+    # 4 good (<5% inaud),
+    # 3 fair (<20% inaud),
+    # 2 usable (>20% inaud but transcript available),
+    # 1 bad (db < 40 so not sent for transcription),
+    # 0 awaiting transcription,
+    # note that interviews missing from audio QC due to SOP violations or other issues will not be reflected here at all!
+    # these counts relate only to interviews that were able to be processed by QC
+    score=[]
+    for x,y in zipped_list:
+
+        if np.isnan(x) and y > 40:
+            _score=0
+        elif np.isnan(x):
+            _score=1
+        elif x < 0.01:
+            _score=5
+        elif x < 0.05:
+            _score=4
+        elif x < 0.2:
+            _score=3
+        else:
+            _score=2
+        
+        score.append(_score)
+        
+    return score
```

Comment:
```python
psychs_only["audio_quality_category"] = [0 if np.isnan(x) and y > 40 else (5 if np.isnan(x) else (1 if x < 0.01 else (2 if x < 0.05 else (3 if x < 0.2 else 4)))) for x,y in zip(psychs_only["inaudible_per_word"].tolist(),psychs_only["overall_db"].tolist())]
```

This was Michaela's original code. I found it to be complex for our successors to understand. So I simplified it.

Reply:
Also, as the goal of this PR states, I reversed the scores. So more number means better quality now and vice-versa. On the other hand, score `0` has been kept the same.